### DISCUSSION TITLE
This is awkward

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [theforumhelpers/theforumhelpers.github.io#269] [General]
# Well, In short...

 the studio description doesn't have any applications form..

## In long:

I went to the studio, it says go to this website. I went to the website, it says u need 500 posts, now go back to studio description and fill our form (seriously? This is recursive lol). Now I went back to description but didn't find a form xD 

### Long method:
I know... the owner of TFH is gone. So just make an applications form in  the website, it will encode and give the coded text, then we can come to on the studio, members can decode and see if application is applicable and do whatever.

### Short method:
Type out the format in the website and say to go comment in the studio. OR
Make a form (like google firms or Google form itself) and let people submit applications. Members can see through website itself 
### Waste method:
Shift studios xP or make a new studio/user

Ohh.. there is a PR already lol https://github.com/theforumhelpers/theforumhelpers.github.io/pull/267
There is an issue too? -_- https://github.com/theforumhelpers/theforumhelpers.github.io/issues/264
I'm not gonna delete this though.. so you people can read my ideas

**Comment by Quantum-Codes** [comment]
How do I make a heading big?

**Reply by BarelySmooth** [reply]
Use `## Big heading`
## Big heading

**Comment by BarelySmooth** [comment]
PR #267 will solve this, I think applications are closed until then.

**Reply by Quantum-Codes** [reply]
Ty

**Comment by aidan-crockett** [comment]
We're still working on adding an applications page. Don't worry, the recursion will end. XD

**Reply by Quantum-Codes** [reply]
XD

**Reply by Quantum-Codes** [reply]
Ended!!!!
